vividus-plugin-lighthouse/src/main/java/org/vividus/lighthouse/model/PerformanceScoreSelector.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.lighthouse.model;
+
+import static org.vividus.lighthouse.utils.LighthouseResultUtils.getScore;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import com.google.api.services.pagespeedonline.v5.model.LighthouseResultV5;
+
+public enum PerformanceScoreSelector
+{
+    HIGHEST(r -> sortAndGet(r, r.size() - 1)),
+    MID(r -> sortAndGet(r, r.size() / 2)),
```

Comment:
shouldn't it be average? what's the point to have some middle value instead of average?